src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/BreakpointComplications.java

```diff
@@ -176,12 +176,12 @@ boolean hasDupSeqButNoStrandSwitch() {
     BreakpointComplications(final ChimericAlignment chimericAlignment, final byte[] contigSeq) {
 
         // if-else testing order matters here:
-        // if the two segments in the input chimeric alignment suggests simple translocation {@see ChimericAlignment#isNotSimpleTranslocation()}, then
+        // if the two segments in the input chimeric alignment suggests simple translocation {@see ChimericAlignment#isLikelySimpleTranslocation()}, then
         //  we need other types of evidence to fully resolve type of event, until then we can only say a novel adjacency (BND) is detected;
         // if the two segments in the input chimeric alignment map to the same chromosome, then
         //  the segment with lower coordinate on the locally-assembled contig could map to a higher reference coordinate region
         //  under two basic types of SV's: inversion (strand switch necessary) and "translocation" (no strand switch necessary)
-        final boolean suggestsSimpleTranslocation = !chimericAlignment.isNotSimpleTranslocation();
+        final boolean suggestsSimpleTranslocation = chimericAlignment.isLikelySimpleTranslocation();
```

Comment:
logically, isNotSimpleTranslocation is not the same as the negation of isLikelySimpleTranslocation, so we are using the wrong criterion here to resolve complications.